.github/workflows/ci.yml

```diff
@@ -446,8 +443,8 @@ jobs:
         run: |
           # Generate a valid artifact name and make it available to next steps as
           # an environment variable ARTIFACT_NAME
-          # ", :, <, >, |, *, ?, \, / are not allowed in artifact names but we only use : so we remove it
-          name=$(echo -n "${{ matrix.modules }}" | sed -e 's/[:]//g')
+          # ", :, <, >, |, *, ?, \, / are not allowed in artifact names, replace it with an underscore
```

Comment:
I had to switch from using short module names to full names with forward slashes, to be able to do a join with the impacted modules list produced by GIB. Instead of adding just one more character, I added all of them.

The job name suggests this sanitizes all names, not only currently used ones. The comment listed all invalid characters, but the command only fixed one of them. This is a cognivite overhead, where you'd have to check if this expression is correct every time when changing `modules` in the matrix.